pi_mqtt_gpio/server.py

```diff
@@ -641,7 +647,75 @@ def gpio_interrupt_callback(module, pin):
     )
 
 
+def hass_annonce_digital_input(in_conf, topic_prefix, mqtt_config):
```

Comment:
typo in 'announce' here, as well as on line 683 (and the usages of the functions)

Reply:
fixed